From 990bc7c8a234a0f446a7d56067b1a9f8348e0f35 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 16 Jul 2020 18:19:58 +0100
Subject: [PATCH] ARM: dts: Select the actpwr LED trigger on Zeroes

The new "actpwr" LED trigger guarantees that the combined PWR and ACT
LED is on for at least half of every second. Under heavy SD card
load it emits a steady 1Hz square wave.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/bcm2708-rpi-zero-w.dts | 4 ++--
 arch/arm/boot/dts/bcm2708-rpi-zero.dts   | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

--- a/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-zero-w.dts
@@ -144,8 +144,8 @@
 &leds {
 	act_led: act {
 		label = "led0";
-		linux,default-trigger = "mmc0";
-		gpios = <&gpio 47 0>;
+		linux,default-trigger = "actpwr";
+		gpios = <&gpio 47 GPIO_ACTIVE_LOW>;
 	};
 };
 
--- a/arch/arm/boot/dts/bcm2708-rpi-zero.dts
+++ b/arch/arm/boot/dts/bcm2708-rpi-zero.dts
@@ -98,8 +98,8 @@
 &leds {
 	act_led: act {
 		label = "led0";
-		linux,default-trigger = "mmc0";
-		gpios = <&gpio 47 0>;
+		linux,default-trigger = "actpwr";
+		gpios = <&gpio 47 GPIO_ACTIVE_LOW>;
 	};
 };
 
